mobile/library/common/jni/jni_interface.cc

```diff
@@ -242,7 +244,7 @@ static void* jvm_on_headers(const char* method, const Envoy::Types::ManagedEnvoy
                                          end_stream ? JNI_TRUE : JNI_FALSE, j_stream_intel);
   // TODO(Augustyniak): Pass the name of the filter in here so that we can instrument the origin of
   // the JNI exception better.
```

Comment:
What I've done in similar situation is to include a `detail` param that is some arbitrary string that describes the context in which this exception is being handled in

Reply:
I like the idea - implemented

Reply:
implemented ur suggestion 👍